workers/unity/Packages/com.improbable.gdk.core/Utility/PlayerLoopUtils.cs

```diff
@@ -0,0 +1,238 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.Experimental.LowLevel;
+using UnityEngine.Experimental.PlayerLoop;
+
+namespace Improbable.Gdk.Core
+{
+    public static class PlayerLoopUtils
```

Comment:
Can this be internal?

Reply:
It could be, but I rather have this public. Since its the only way to use multiple ECS wolds in a decent fashion. It also contains a Print function for debugging.

Reply:
Users will need this if they want to add an ECS world that is not a spatial one.

Reply:
I strongly feel this should _not_ be a public API